frontend2/src/contexts/CurrentTeamProvider.tsx

```diff
@@ -19,6 +19,11 @@ export const CurrentTeamProvider: React.FC<{ children: React.ReactNode }> = ({
   const { authState } = useCurrentUser();
   const { episodeId } = useEpisodeId();
 
+  const leaveMyTeam = useCallback(async (): Promise<void> => {
+    await leaveTeam(episodeId);
```

Comment:
Should this callback have `episodeId` as a dependency?

Reply:
Yes! thanks for catching this